packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -0,0 +1,151 @@
+
+/// The media types that require a user gesture to begin playing.
+///
+/// See https://developer.apple.com/documentation/webkit/wkaudiovisualmediatypes?language=objc.
+enum AudiovisualMediaType {
+  /// No media types require a user gesture to begin playing.
+  ///
+  /// See https://developer.apple.com/documentation/webkit/wkaudiovisualmediatypes/wkaudiovisualmediatypenone?language=objc.
+  none,
+
+  /// Media types that contain audio require a user gesture to begin playing.
+  ///
+  /// See https://developer.apple.com/documentation/webkit/wkaudiovisualmediatypes/wkaudiovisualmediatypeaudio?language=objc.
+  audio,
+
+  /// Media types that contain video require a user gesture to begin playing.
+  ///
+  /// See https://developer.apple.com/documentation/webkit/wkaudiovisualmediatypes/wkaudiovisualmediatypevideo?language=objc.
+  video,
+
+  /// All media types require a user gesture to begin playing.
+  ///
+  /// See https://developer.apple.com/documentation/webkit/wkaudiovisualmediatypes/wkaudiovisualmediatypeall?language=objc.
+  all,
+}
+
+/// A collection of properties that you use to initialize a web view.
+///
+/// A [WebViewConfiguration] object provides information about how to configure
+/// a [WebView] object. Use your configuration object to specify:
+///
+/// * The initial cookies to make available to your web content
+/// * Handlers for any custom URL schemes your web content uses
+/// * Settings for how to handle media content
+/// * Information about how to manage selections within the web view
+/// * Custom scripts to inject into the webpage
+/// * Custom rules that determine how to render content
+///
+/// You create a [WebViewConfiguration] object in your code, configure its
+/// properties, and pass it to the initializer of your [WebView] object. The web
+/// view incorporates your configuration settings only at creation time; you
+/// cannot change those settings dynamically later.
+class WebViewConfiguration {
+  /// Indicates whether HTML5 videos play inline or use the native full-screen controller.
+  set allowsInlineMediaPlayback(bool allow) {
+    throw UnimplementedError();
+  }
+
+  /// The media types that require a user gesture to begin playing.
+  ///
+  /// Use [AudiovisualMediaType.none] to indicate that no user gestures are
+  /// required to begin playing media.
+  set mediaTypesRequiringUserActionForPlayback(
+    Set<AudiovisualMediaType> types,
+  ) {
+    assert(types.isNotEmpty);
+    throw UnimplementedError();
+  }
+
+  /// Indicates whether HTML5 videos require the user to start playing them.
+  ///
+  /// Deprecated for iOS >= 10.0.
+  ///
+  /// Passing false indicates the videos can play automatically.
+  @Deprecated(
+    'Deprecated for iOS 10.0+. '
+    'Please use WebViewConfiguration.mediaTypesRequiringUserActionForPlayback.',
+  )
+  set requiresUserActionForMediaPlayback(bool required) {
+    throw UnimplementedError();
+  }
+
+  /// Deprecated for iOS >= 9.0.
```

Comment:
That's all we support now, so this is dead code. We just never cleaned up the implementation after changing the min version (I guess we missed it in the scrub).